doc/languages-frameworks/vim.section.md

```diff
@@ -153,55 +170,63 @@ Some plugins require overrides in order to function properly. Overrides are plac
+instead add them to the bottom of
+[`overrides.nix`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/overrides.nix).
+If plugin is packaged on luarocks, packaging it through `luaPackages` should be
+preferred.
+
+Finally, there are some plugins that are also packaged in nodePackages because
+they have Javascript-related build steps, such as running webpack. Those
+plugins are not listed in `vim-plugin-names` or managed by `vimPluginsUpdater`
+at all, and are included separately in `overrides.nix`. Currently, all these
+plugins are related to the `coc.nvim` ecosystem of the Language Server Protocol
+integration with Vim/Neovim.
 
 ## Updating plugins in nixpkgs {#updating-plugins-in-nixpkgs}
 
-Run the update script with a GitHub API token that has at least `public_repo` access. Running the script without the token is likely to result in rate-limiting (429 errors). For steps on creating an API token, please refer to [GitHub's token documentation](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).
+Run the update script with a GitHub API token that has at least `public_repo`
+access. Running the script without the token is likely to result in
+rate-limiting (429 errors). For steps on creating an API token, please refer to
+[GitHub's token
+documentation](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).
 
 ```sh
-nix-shell -p vimPluginsUpdater --run 'vim-plugins-updater --github-token=mytoken' # or set GITHUB_API_TOKEN environment variable
+GITHUB_TOKEN=mytoken nix-shell -p vimPluginsUpdater --run 'vim-plugins-updater'
 ```
 
-Alternatively, set the number of processes to a lower count to avoid rate-limiting.
+Alternatively, set the number of processes to a lower count to avoid
+rate-limiting.
 
 ```sh
-nix-shell -p vimPluginsUpdater --run 'vim-plugins-updater --proc 1'
+nix-shell -p vimPluginsUpdater --run 'vim-plugins-updater --jobs 1'
 ```
 
-If you want to update only certain plugins, you can specify them after the `update` command. Note that you must use the same plugin names as the `pkgs/applications/editors/vim/plugins/vim-plugin-names` file.
+If you want to update only certain plugins, you can specify them after the
+`update` command. Note that you must use the same plugin names as the
+`pkgs/applications/editors/vim/plugins/input.csv` file.
 
 ```sh
-nix-shell -p vimPluginsUpdater --run 'vim-plugins-updater update "nvim-treesitter" "LazyVim"'
-```
-
-## How to maintain an out-of-tree overlay of vim plugins ? {#vim-out-of-tree-overlays}
-
-You can use the updater script to generate basic packages out of a custom vim
-plugin list:
-
-```
-nix-shell -p vimPluginsUpdater --run vim-plugins-updater -i vim-plugin-names -o generated.nix --no-commit
+nix-shell -p vimPluginsUpdater --run 'vim-plugins-updater update "https://github.com/folke/lazy.nvim"'
```

Comment:
Does this no longer support by plugin name? Seems more appropriate than having to enter an entire url everytime

Reply:
It uses the same system as `add` command. I plan to remove `https://github.com/` that is required for every plugin in the near future

Reply:
Hmm... alright :S Not a fun regression, but we rarely do individual updates anyways, I suppose.